sos/plugins/openstack_ironic.py

```diff
@@ -122,8 +125,18 @@ class RedHatIronic(OpenStackIronic, RedHatPlugin):
         'openstack-ironic-discoverd-ramdisk'
     ]
 
+    def collect_introspection_data(self):
+        uuids_result = self.call_ext_prog('openstack baremetal node list '
```

Comment:
Output of `openstack baremetal node list` is not required to collect, am I right? (it wont be when you use `call_ext_prog`)

Reply:
It's collected in the parent class, so it does not make sense to collect it here again.

Reply:
Hm.. can't we merge the two calls into one, then? (definitely there can be some trade-off in case that command execution is fast every time).

Reply:
This is how the two outputs look like: http://paste.openstack.org/show/627601/. Actually, the former is MUCH longer, but I did not use --long here. To get the latter we need to parse this table in Python, which is probably on overkill.

The command itself is one SELECT from the database.

Reply:
That sounds acceptable to me to have one extra call, then 9if we would have to parse the output from --long output otherwise). Thanks for explanation.